src/main/kotlin/de/bund/digitalservice/useid/identification/MockDatasource.kt

```diff
@@ -43,4 +47,12 @@ class MockDatasource {
         session.eIDSessionId = eIDSessionId
         log.info { "set new eIDSessionId: ${session.eIDSessionId} for session with useIDSessionId: $useIDSessionId" }
     }
+
+    fun delete(session: IdentificationSession): Boolean {
```

Comment:
why do we return a `Boolean`? will the return value be consumed in the Controller? would it be ok to return `Unit` instead ?

Reply:
What is unit about? I don't know that... 
I've also thought about throwing an exception when the element could not be removed. What you think? (So we would remove the return type entirely)